core/templates/services/improvements.service.ts

```diff
@@ -29,6 +29,9 @@ export class ImprovementsService {
   INTERACTION_IDS_REQUIRED_TO_BE_RESOLVED = ['TextInput'];
 
   isStateForcedToResolveOutstandingUnaddressedAnswers(state: State): boolean {
+    if (!state || (state.interaction.id === null)) {
+      return false;
+    }
     return !!state && this.INTERACTION_IDS_REQUIRED_TO_BE_RESOLVED.indexOf(
```

Comment:
If you're checking for !state above, why do you still need !!state here?

Reply:
Removed!